### PR TITLE
Revert "Update HTTPS and Storage Explorer instructions. (#431)"

### PR DESCRIPTION
**Pull request by lubas569**
Reverts Azure/Azurite#435

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -310,7 +312,7 @@ In basic level, `--oauth basic`, Azurite will do basic authentication, like vali
 
 ### Command Line Options Differences between Azurite V2
 
-Azurite V3 supports SharedKey, Account Shared Access Signature (SAS), Service SAS, OAuth, and Public Container Access authentications, you can use any Azure Storage SDKs or tools like Storage Explorer to connect Azurite V3 with any authentication strategy.
```
Comment (blueww) [inline README.md:313]: Why remove oauth? Oauth us supported in the latest release.

File: README.md [hunk 2]
```diff
@@ -479,7 +375,7 @@ Azurite V3 provides support for a default storage account as General Storage Acc
 - Account name: `devstoreaccount1`
 - Account key: `Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==`
 
-> Note. Besides SharedKey authentication, Azurite V3 supports account, OAuth, and service SAS authentication. Anonymous access is also available when container is set to allow public access.
```
Comment (blueww) [inline README.md:482]: why remove oauth?

File: README.md [hunk 3]
```diff
@@ -364,112 +366,6 @@ This feature is in preview, when Azurite changes database table schema, you need
 
 > Tips. Create database instance quickly with docker, for example `docker run --name mysql -p 3306:3306 -e MYSQL_ROOT_PASSWORD=my-secret-pw -d mysql:latest`. Grant external access and create database `azurite_blob` using `docker exec mysql mysql -u root -pmy-secret-pw -e "GRANT ALL PRIVILEGES ON *.* TO 'root'@'%' WITH GRANT OPTION; FLUSH PRIVILEGES; create database azurite_blob;"`. Notice that, above commands are examples, you need to carefully define the access permissions in your production environment.
 
-## HTTPS Setup
```
Comment (blueww) [inline README.md:367]: Why remove this section?

**Comment by blueww** [comment]
@XiaoningLiu 
Would you please help to review the readme change?
There are many changes.

**Comment by XiaoningLiu** [comment]
This PR doesn't make sense to revert readme expected changes.

**Pull request closed without merging**